dot_lpdev_cmds.sh

```diff
@@ -682,12 +701,85 @@ function __lpdev_node_transcoder {
   echo "Requesting test tokens"
   curl -X "POST" http://localhost:$transcoderApiPort/requestTokens
 
+  echo "Initializing current round"
+  curl -X "POST" http://localhost:$transcoderApiPort/initializeRound
+
   echo "Activating transcoder"
-  curl -X "POST" http://localhost:$transcoderApiPort/activateTranscoder\
-    --data-urlencode "blockRewardCut=10&feeShare=5&pricePerSegment=1&amount=500"
+  curl -d "blockRewardCut=10&feeShare=5&pricePerSegment=1&amount=500" \
+    -H "Content-Type: application/x-www-form-urlencoded" \
+    -X "POST" http://localhost:$transcoderApiPort/activateTranscoder\
 
 }
 
+function __lpdev_verifier_init {
+  if [ -d $srcDir/verification-computation-solver ]
+  then
+    echo "Verifier src directory exists.  Trying to pull the latest version."
+    git pull
+  else
+    echo "Cloning github.com/livepeer/verification-computation-solver into src directory"
+    OPWD=$PWD
+    cd $srcDir
+    git clone "https://github.com/livepeer/verification-computation-solver.git"
+    cd $OPWD
+  fi
+
+  ##
+  # Update npm
+  ##
+
+  listModules=($(ls $srcDir/verification-computation-solver/node_modules))
+  if [ -d $srcDir/verification-computation-solver/node_modules ] && [ ${#listModules[@]} -gt 0 ]
+  then
+    echo "Npm packages already installed"
+  else
+    echo "Running \`npm install\`"
+    OPWD=$PWD
+    cd $srcDir/verification-computation-solver
+    npm install
+    cd $OPWD
+  fi
+}
+
+function __lpdev_verifier {
+  __lpdev_verifier_init
+  __lpdev_ipfs_init
+
+  echo "Making verifier address"
```

Comment:
This would create a new node process even if one is already running, is that the intended behavior? Should we exit early if `verifierPid` is set?

Reply:
Yep.  That makes sense.